junit5-extension/src/test/java/io/kroxylicious/testing/kafka/junit5ext/TemplateTest.java

```diff
@@ -124,10 +127,14 @@ void afterAll() {
 
     @SuppressWarnings("unused")
     static Stream<List<Annotation>> tuples() {
-        return Stream.of(
+        var tuples = new ArrayList<>(List.of(
                 List.of(brokerCluster(1), kraftCluster(1)),
-                List.of(brokerCluster(3), kraftCluster(1)),
-                List.of(brokerCluster(3), zooKeeperCluster()));
+                List.of(brokerCluster(3), kraftCluster(1))));
+        if (ZOOKEEPER_AVAILABLE) {
```

Comment:
its an annoying pattern but I haven't thought of any brighter ideas.

Reply:
Agreed, I didn't much like the code, but it seemed like the only way.